utils/src/main/scala/za/co/absa/enceladus/utils/error/UDFLibrary.scala

```diff
@@ -46,10 +44,11 @@ case class UDFLibrary()(implicit val spark: SparkSession) {
     ErrorMessage.confNegErr(errCol, rawValue)
   })
 
-  spark.udf.register("arrayDistinctErrors", (arr: mutable.WrappedArray[ErrorMessage]) => arr.distinct.filter((a: AnyRef) => a != null))
+  spark.udf.register("arrayDistinctErrors",
+    (arr: mutable.WrappedArray[ErrorMessage]) => arr.distinct.filter((a: AnyRef) => a != null))
 
-  val cleanErrCol = new UDF1[Seq[Row], Seq[Row]] {
-    override def call(t1: Seq[Row]) = {
+  private val cleanErrCol: UDF1[Seq[Row], Seq[Row]] = new UDF1[Seq[Row], Seq[Row]] {
```

Comment:
I'd leave this up to type inference, since on the right hand side we have the exact same type information with `new UDF1[Seq[Row], Seq[Row]]`